app/(routes)/video-consultation-form/components/StripeElementsStep.tsx

```diff
@@ -34,11 +34,19 @@ function PaymentForm( { price }: { price: number } ) {
         },
         redirect: 'if_required',
       });
+
       if (confirmError) {
         setError(confirmError.message || 'Payment failed');
         sendToParent({ type: 'payment_error', data: { error: confirmError.message } });
         return;
       }
+    } catch {
+      setError('Payment failed! Please try again.');
+      setIsProcessing(false);
+      return;
+    }
```

Comment:
Added lots of more specific try/catch statements to improve error messages and only error on critical failures.